test/service/test_dss_proxy.py

```diff
@@ -34,17 +34,15 @@ def test_dss_files_proxy(self):
                                              headers={'Location': dss_url + token,
                                                       'Retry-After': str(retry_after)}))
             azul_url = f'{self.base_url}/dss/files/{file_uuid}?replica=aws&version={file_version}'
-            before = time.time()
             response = requests.get(azul_url, allow_redirects=False)
-            request_duration = time.time() - before
-            self.assertLessEqual(retry_after, request_duration)
-            self.assertLess(request_duration, 2 * retry_after)
-            self.assertEqual(301, response.status_code)
-            self.assertEqual(azul_url + token, response.headers['Location'])
+            self.assertEqual(200, response.status_code)
+            json_response = json.loads(response.content)
```

Comment:
```suggestion
            body = json.loads(response.content)
```

Reply:
Why is there no assertion on body['Retry-After'] and body['Status']?

I shouldn't have to catch these things. They drain my "mental energy" which is needed to solve bigger problems. Please try to be more meticulous.